build.gradle

```diff
@@ -87,6 +88,24 @@ checkstyle {
     configFile file("configs/checkstyle/checkstyle.xml")
 }
 
+apply plugin: 'com.diffplug.spotless'
+
+spotless {
+    java {
+        licenseHeaderFile(file('license-header'))
+
+        // only apply formatting rules to updated files
+        // ratchetFrom 'origin/master'
+
+        // googleJavaFormat()
+        // importOrder()
+        // removeUnusedImports()
```

Comment:
Should we also add `trimTrailingWhitespace` and `endWithNewline` in the follow up PR?

Reply:
Makes sense, I'll add it to the next PR.

Reply:
I added these options (commented out) in the rebased commit.